package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "digital-analytics-program",
+  "version": "4.0.0",
+  "description": "Provides a JavaScript file for US federal agencies to link or embed in their websites to participate in the Digital Analytics Program.",
+  "main": "Universal-Federated-Analytics.js",
+  "scripts": {
+    "build": "npx uglifyjs Universal-Federated-Analytics.js --output Universal-Federated-Analytics-Min.js",
+    "clean": "rm ./Universal-Federated-Analytics-Min.js",
+    "lint": "npx eslint Universal-Federated-Analytics.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digital-analytics-program/gov-wide-code.git"
+  },
+  "license": "public domain",
```

Comment:
The code isn't public domain - right now the only indication of a license is "Copyright 2017 by E-Nor Inc." at https://github.com/digital-analytics-program/gov-wide-code/blob/master/Universal-Federated-Analytics.js#L8. It does not have a visible open source license, so is currently not open source software.

Reply:
(I would just remove the `license` field from the `package.json` for now.)

Reply:
"E-Nor Inc" can have a copyright to government work?